lib/inject/inject_parser.dart

```diff
@@ -72,7 +72,7 @@ class DartPadInjectException implements Exception {
   final String message;
   DartPadInjectException(this.message);
   @override
-  String toString() => '$message';
```

Comment:
I suspect this was for casting null to the actual string 'null'. I'd hold off changing it until after the null safe conversion

Reply:
I've ignored this lint for now to be safe. :)

Reply:
Null safety shakes a lot of these things out into the light of day 😎